core/sds.c

```diff
@@ -2307,10 +2305,16 @@ oio_sds_get_content_properties(struct oio_sds_s *sds, struct oio_url_s *url,
 	err = (struct oio_error_s*) oio_proxy_call_content_get_properties(sds->h, url, &value);
 	if (err)
 		return err;
-
 	json_object *json = json_tokener_parse(value->str);
 	json_object_object_foreach(json,key,val) {
-		fct(ctx, key, json_object_get_string(val));
+		(void) key;
+		const gchar *json_string = json_object_get_string(val);
+		json_object *json_properties = json_tokener_parse(json_string);
```

Comment:
Is that really necessary to cast the object to a string and parse it again as a json object? You can probably use `json_object_object_foreach` directly on `val`.
It would also be a good idea to check `key`.